### PR TITLE
Update ScoreToLetterGradeTest.java and request for review

### PR DESCRIPTION
**Pull request by DevelopnTest**
remove one line

### CODE REVIEW

File: LetterGrade/src/ScoreToLetterGradeTest.java [hunk 1]
```diff
@@ -15,7 +15,6 @@ protected void tearDown() throws Exception {
 	public void testLetterGrade(){
 		assertEquals(ScoreToLetterGrade.letterGrade(100),'A');
 		assertEquals(ScoreToLetterGrade.letterGrade(90),'A');
-		assertEquals(ScoreToLetterGrade.letterGrade(89),'B');
```
Comment (cxl1521) [inline LetterGrade/src/ScoreToLetterGradeTest.java]: **D
Duplicate of #evelopnTest** Try to add another test method

> assertEquals(ScoreToLetterGrade.letterGrade(X), 'X');

Comment (cxl1521) [inline LetterGrade/src/ScoreToLetterGradeTest.java]: **DevelopnTest** add

>assertEquals(ScoreToLetterGrade.letterGrade(-1), 'X');

File: LetterGrade/src/ScoreToLetterGradeTest.java [hunk 2]
```diff
@@ -15,7 +15,6 @@ protected void tearDown() throws Exception {
 	public void testLetterGrade(){
 		assertEquals(ScoreToLetterGrade.letterGrade(100),'A');
 		assertEquals(ScoreToLetterGrade.letterGrade(90),'A');
-		assertEquals(ScoreToLetterGrade.letterGrade(89),'B');
 		assertEquals(ScoreToLetterGrade.letterGrade(80),'B');
 		assertEquals(ScoreToLetterGrade.letterGrade(70),'C');
```
Comment (cxl1521) [inline LetterGrade/src/ScoreToLetterGradeTest.java]: remove this line

File: LetterGrade/src/ScoreToLetterGradeTest.java (hunk 1 above)
Comment (DevelopnTest) [inline LetterGrade/src/ScoreToLetterGradeTest.java]: ok

Comment (DevelopnTest) [reply LetterGrade/src/ScoreToLetterGradeTest.java]: ok

Comment (DevelopnTest) [reply LetterGrade/src/ScoreToLetterGradeTest.java]: will do it

**Comment by DevelopnTest** [comment]
the change the test methods have been done

**Pull request merged**